salt/states/junos.py

```diff
@@ -71,12 +79,19 @@ def rpc(name, dest=None, format="xml", args=None, **kwargs):
               Name of the interface whose information you want.
     """
     ret = {"name": name, "changes": {}, "result": True, "comment": ""}
+    if rpc is None:
+        # for backward, will use name as rpc if rpc not provided explicitly
+        # for example
+        # get-interface-information:
+        #     junos.rpc:
+        #         - dest: /home/user/rpc.log
+        rpc = name
     if args is not None:
         ret["changes"] = __salt__["junos.rpc"](
-            name, dest=dest, format=format, args=args, **kwargs
+            rpc, dest=dest, format=format, args=args, **kwargs
```

Comment:
see above comment about new parameters best placed at the end

Reply:
reverted via https://github.com/saltstack/salt/pull/57281/commits/682cb0a08baa14e1fa4684163c5588cbda672d25#diff-442286d51776ddefc791bc7f15e42fc2R80